src/community/geofence-server/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- set parent pom to community pom -->
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>community</artifactId>
+        <version>2.8-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.geoserver.community</groupId>
+    <artifactId>gs-geofence-server</artifactId>
+    <packaging>jar</packaging>
+    <version>2.8-SNAPSHOT</version>
+    <name>GeoFence security internal server</name>
+
+    <properties>
+        <gf.version>3.0-SNAPSHOT</gf.version>
+        <hibernate-version>3.6.0.Final</hibernate-version>
+        <hibernate-generic-dao-version>1.1.0</hibernate-generic-dao-version>
+        <hibernate-spatial-version>1.1.1</hibernate-spatial-version>
+        <hibernate-spatial-h2-version>1.1.1</hibernate-spatial-h2-version>
+    </properties>
```

Comment:
I am not sure about listing these here - usual practice is the root pom.xml

Perhaps this is okay as it is a community module, I would just hate to see two community modules conflict.